src/snyk/common/configuration/configuration.ts

```diff
@@ -292,10 +280,6 @@ export class Configuration implements IConfiguration {
     return Configuration.source;
   }
 
-  get baseApiUrl(): string {
```

Comment:
This was added in https://github.com/snyk/vscode-extension/commit/213c781f9debe959695206e8ea2350c832df2658 but it seems the usage has since been removed